libs/core/langchain_core/messages/ai.py

```diff
@@ -152,8 +159,28 @@ def lc_attributes(self) -> Dict:
     @root_validator()
     def init_tool_calls(cls, values: dict) -> dict:
         if not values["tool_call_chunks"]:
-            values["tool_calls"] = []
-            values["invalid_tool_calls"] = []
+            if values["tool_calls"]:
```

Comment:
cc @ccurme changing init behavior of chunks to create chunks from tool calls if only the latter is present

Reply:
should add unit test